_posts/2022-06-28-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+We are pleased to announce the upcoming 2022 Trino Summit. The event is
+scheduled as *hybrid* event on the 10th of November 2022. You'll be able to join
+us online, or you can make the trip to San Francisco and meet us at the
+Commonwealth Club on the downtown waterfront.
+
+The event will be hosted by Starburst, one of the largest contributors to Trino.
+[Register now](https://www.starburst.io/info/trinosummit/) to attend (online
```

Comment:
Can we make this a big register button here or in addition on the full page?

Reply:
Or maybe use a highlight as in https://trino.io/blog/2021/04/21/the-definitive-guide.html .. we just need to make the register action big and a central point

and maybe do another button for submit a talk

Reply:
Look at the source for other pages to find markup for a button link (e.g. download.md)